status-im/status-react
# [#21081] Privacy Policy and metrics modal in settings while sharing usage data

fixes #21081

### Summary

From acceptance criteria in the issue description:


:white_check_mark: When enabling the setting Privacy and security > Share usage data with Status, the user should always see the drawer confirmation.

https://github.com/user-attachments/assets/71b4597b-c9d6-409f-99fd-b4437447d95f



:white_check_mark: We should give the user the in-app link to the Privacy Policy, just as we do in the onboarding.


https://github.com/user-attachments/assets/dcef66db-8c27-4507-bb04-8b4e2420c5c7




:white_check_mark: Update resources/privacy.mdwn to match exactly https://docs.google.com/document/d/1J4w2q6-fdVkXyC4D0aaQMQPq1rhfCCiz4I8p5gVUVds
:white_check_mark:  Update resources/terms-of-use.mdwn to match exactly https://docs.google.com/document/d/1JIhxYhB5aVCzq6acD7TSpvcVg1p37C9H/edit

https://github.com/user-attachments/assets/8440f254-66f4-45c5-99d2-5dc37d1767e9



:white_check_mark: This PR also adds the "From all profiles on device" text:

![image](https://github.com/user-attachments/assets/3b94cb84-a7c4-4d03-9b60-6adf48515ec9)


### Review notes

[Link to Figma, just for reference](https://www.figma.com/design/JlpPhJp0SMnEyBQy4nOZHo/Settings-for-Mobile?node-id=8202-7377&t=K9hZhGdiGZuWklga-4)


#### Platforms

- Android
- iOS

### Steps to test

- Open Status in a fresh install
- Check the terms of use and privacy policy links
- Create a new profile
- Navigate to the Profile Settings > Privacy and Security > test the toggle

status: ready
